lib/article_detail.dart

```diff
@@ -26,10 +27,10 @@ class _DetailScreenState extends State<DetailScreen> {
 
   @override
   void initState() {
-    _shareMessage = "Check out this article '${widget.post.title}'\n${widget.post.link}";
+    _shareMessage =
+        "Check out this article '${widget.post.title}'\n${widget.post.link}";
```

Comment:
@alexdivadi  in thi project it is ok, but for future reference try to make whitespce changes all in one mr if you're going to do it. i wrote this so long ago... so this makes it look like you made changes but it's just whitespce. another reason we'll get taht sorted from the beginning in new project.